src/main/webapp/script.js

```diff
@@ -212,6 +202,29 @@ function enableItemSubmissionIfUnique(submitButton, itemCard, omdbItem) {
       });
 }
 
+/**
+ * Enables access to the profile if the user is logged in by adding a "Profile" link to the navbar,
+ * if the user is not logged in then it adds a link to login.
+ */
+function enableProfileSectionIfLoggedIn() {
```

Comment:
Nit: the name of the function makes it seem like it doesn't do anything if the user is not logged in.

Can we rename to something like `setUpNavBarProfileSection`?

Reply:
Yeah the last name might've been a little confusing, I changed it in last commit.